src/helpers.coffee

```diff
@@ -268,6 +268,20 @@ exports.nameWhitespaceCharacter = (string) ->
     when '\t' then 'tab'
     else string
 
+exports.parseNumber = (string) ->
```

Comment:
Per other [comment](https://github.com/jashkenas/coffeescript/pull/5269#issuecomment-569542323), extracted `parseNumber()` helper so that it can be used instead of `Number()` in the multiple places in `nodes.coffee` where `Number()` was being used